docs/serverless/03-02-api-rules.md

```diff
@@ -13,6 +13,8 @@ To access a Function within the cluster, use the `{function-name}.{namespace}.sv
 
 1. The API Gateway Controller creates an Istio Virtual Service and Access Rules according to details specified in the CR. Such a Function service is available under the `{host-name}.{domain}` endpoint, such as `my-function.kyma.local`.
 
+> **NOTE:** If you are using Minikube, before you can access the function you have to add the endpoint to the Minikube IP entry in /etc/hosts file (administrator rights are required to make changes in this file).
```

Comment:
`(administrator rights are required to make changes in this file)` is unnecessary for me.

Reply:
And also I think that this note should be under the Tip (or above).